common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/ProductEntitlementMappings.kt

```diff
@@ -0,0 +1,30 @@
+package com.revenuecat.purchases.common.offlineentitlements
+
+import org.json.JSONObject
+
+data class ProductEntitlementMappings(
```

Comment:
This name differs from iOS, but I think it represents the object better. Lmk what you think, I can change it back easily

Reply:
I'm gonna rename it in iOS 👍🏻